doc/setup_iothub.md

```diff
@@ -6,55 +6,12 @@ Before you can communicate with IoT Hub from a device you must create an IoT hub
 
 ## Create an IoT hub
```

Comment:
Same as above: this should be a document on MSDN.

Reply:
The text on the readme redirects to the MSDN document on Iot Hub and DPS